src/config/connection/postgres/connection.ts

```diff
@@ -0,0 +1,19 @@
+import sequelize from 'sequelize';
+import postgresOptions from './config.postgres';
+
+export const connection: sequelize.Sequelize = new sequelize.Sequelize(
```

Comment:
It is ok to use typings for variables, but better approach is to use typings for input and output functions. Here you use sequelize method and don't need extra typings at all.